crypto/cipher/aes_gcm_nss.c

```diff
@@ -387,11 +394,22 @@ static srtp_err_status_t srtp_aes_gcm_nss_get_tag(void *cv,
  *	enc_len	length of encrypt buffer
  */
 static srtp_err_status_t srtp_aes_gcm_nss_decrypt(void *cv,
-                                                  uint8_t *buf,
-                                                  size_t *enc_len)
+                                                  const uint8_t *src,
+                                                  size_t src_len,
+                                                  uint8_t *dst,
+                                                  size_t *dst_len)
 {
-    srtp_err_status_t status =
-        srtp_aes_gcm_nss_do_crypto(cv, false, buf, enc_len);
+    uint8_t tagbuf[16];
```

Comment:
While no less confusing to me than the `encrypt()` function, it might be good to replicate much of the comments. That said, it would be good if the comments on `encrypt()` would speak about which "NULL buffer" (src or dst). The `When we get a non-NULL buffer` doesn't indicate which buffer.

Reply:
I guess that is an old comment, will update.